### PR TITLE
Updated Compatibility info and GPU info (Vulkan, SPIRV, and Texture Recompression)

### PR DESCRIPTION
**Pull request by Mou-Ikkai**
Noticed that the GPU section of Ryujinx was outdated. Decided to also update the game compatibility information while at it.
Additions to GPU section:
The addition of Vulkan and SPIR-V shaders, along with the new feature of Texture Recompression.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
 
 - **GPU**
 
-  The GPU emulator emulates the Switch's Maxwell GPU using the OpenGL API (version 4.5 minimum) through a custom build of OpenTK. There are currently four graphics enhancements available to the end user in Ryujinx: disk shader caching, resolution scaling, aspect ratio adjustment and anisotropic filtering. These enhancements can be adjusted or toggled as desired in the GUI.
+  The GPU emulator emulates the Switch's Maxwell GPU using the OpenGL API (version 4.5 minimum) through a custom build of OpenTK. As of the end of July 2022, the GPU emulator now supports the usage of the Vulkan API and SPIR-V Shaders. There are currently five graphics enhancements available to the end user in Ryujinx: Disk Shader Caching, Resolution Scaling, Aspect Ratio Adjustment, Anisotropic Filtering, and Texture Recompression. These enhancements can be adjusted or toggled as desired in the GUI.
```
Comment (MutantAura) [inline README.md:93]:
```suggestion
  The GPU emulator emulates the Switch's Maxwell GPU using either the OpenGL (version 4.5 minimum) or Vulkan APIs through a custom build of OpenTK or Silk.NET respectively. There are currently five graphics enhancements available to the end user in Ryujinx: Disk Shader Caching, Resolution Scaling, Aspect Ratio Adjustment, Anisotropic Filtering, and Texture Recompression. These enhancements can be adjusted or toggled as desired in the GUI.
```

As new as Vulkan is the readme should be purely informative on the features available. Not really sure why SPIR-V was mentioned either as it's sort of given by having a Vulkan backend; we don't explicitly state GLSL next to OpenGL afterall.

[the remark this answers is not in the document]
Comment (MutantAura) [reply README.md:93]: Not sure why that came through twice but whatever

**Comment by gdkchan** [comment]
I'm not sure if texture recompression counts as an enhancement.

**Comment by Mou-Ikkai** [comment]
> I'm not sure if texture recompression counts as an enhancement.

It's definitely an enhancement for lower VRam GPUs.

**Comment by EmulationFanatic** [comment]
> > I'm not sure if texture recompression counts as an enhancement.
> 
> It's definitely an enhancement for lower VRam GPUs.

Its more of an optional break/fix workaround though. It does not "enhance" anything except the avoidance of crashes due to inherent limitations on older GPUs.

**Review by CloneDeath — APPROVED** [review]

**Comment by Mou-Ikkai** [comment]
Fixed the ReadME and updated for us now having 3700 tested games!

File: README.md [hunk 2]
```diff
@@ -36,8 +36,8 @@
 
 ## Compatibility
 
-As of October 2022, Ryujinx has been tested on approximately 3,600 titles; over 3,500 boot past menus and into gameplay, with roughly 3,000 of those being considered playable. You can check out the compatibility list [here](https://github.com/Ryujinx/Ryujinx-Games-List/issues).  
-Anyone is free to submit a new game test or update an existing game test entry; simply follow the new issue template and testing guidelines, or post as a reply to the applicable game issue. Use the search function to see if a game has been tested already!
+As of October 2022, Ryujinx has been tested on approximately 3,700 titles; over 3,500 boot past menus and into gameplay, with roughly 3,000 of those being considered playable.
+You can check out the compatibility list [here](https://github.com/Ryujinx/Ryujinx-Games-List/issues). Anyone is free to submit an updated test on an existing game entry; simply follow the new issue template and testing guidelines, or post as a reply to the applicable game issue. Use the search function to see if a game has been tested already!
```
Comment (EmulationFanatic) [inline README.md:40]: Why are we removing the indication that anyone is free to submit a new game test? The new wording makes it seem like the only allowed action is to submit an updated test on an existing entry.
We want new game test entries from as many people as will perform valid tests.

Comment (Mou-Ikkai) [reply README.md:40]: Eh? ..Oh, that was a confusion on my half when looking at the conflict.

**Review by EmulationFanatic — APPROVED** [review]
LGTM

**Pull request merged**